javascript-modal/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.open {
+  background-color: rgb(71, 55, 223);
+  color: white;
+  padding: 5px 25px;
+  border: 1px solid rgb(71, 55, 223);
+  border-radius: 5px;
+  font-size: 20px;
+  position: absolute;
```

Comment:
This button should be in the normal document flow, it shouldn't be absolute positioned. You could absolute or fixed position the modal, that is the content meant to sit on a layer above normal content.